src/components/RelatedArticles.tsx

```diff
@@ -0,0 +1,133 @@
+import Card from "./Card";
+import Grid from "./Grid/web";
+import Text from "./Text";
+import Divider from "./Divider";
+import { imgix, formatDateAbbreviated } from "../utils";
+import { GetArticle, GetArticles } from "../aws";
+
+export function RelatedArticles({
+  article,
+  articles,
+  classNames,
+  theme,
+}: {
+  article: GetArticle;
```

Comment:
Just for better clarity might be good to name this parameter currentArticle or something of that nature. Also, a quick one sentence blurb at the top about this component just to improve documentation.